verification/curator-service/ui/src/components/landing-page/LandingPage.tsx

```diff
@@ -92,8 +98,22 @@ export default function LandingPage({
     const smallHeight = useMediaQuery('(max-height:1050px)');
     const classes = useStyles({ smallHeight });
     const lastLocation = useLastLocation();
+    const location = useLocation<LocationState>();
 
     const [registrationScreenOn, setRegistrationScreenOn] = useState(false);
+    const [changePasswordScreenOn, setChangePasswordScreenOn] = useState(false);
+
+    useEffect(() => {
+        // temporary - to be changed while building api.
```

Comment:
Don't do it inside `useEffect` hook. It would be better to setup new route using `react-router`